homeassistant/components/fints/sensor.py

```diff
@@ -155,6 +156,8 @@ def detect_accounts(self):
             try:
                 self.client.get_holdings(account)
                 holdings_accounts.append(account)
+            except FinTSClientError:
+                pass
```

Comment:
Should we be handling client errors instead of just ignoring them?

Reply:
In this case behavior changed so that the client error is raised instead of the internal dialog error.

Reply:
Sure but that doesn't change the fact that we should handle things, instead of ignoring it.

So this review comment remains.

Reply:
I refactored this part using new API in the library so this control flow by exception handling is not needed anymore.